### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aksm-ms**
#37

### CODE REVIEW

**Review by N-Usha — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ You can also use this Github Action to deploy your customized image into an Azur
 
 For deploying container images to Kubernetes, consider using [Kubernetes deploy](https://github.com/Azure/k8s-deploy) action. This action requires that the cluster context be set earlier in the workflow by using either the [Azure/aks-set-context](https://github.com/Azure/aks-set-context/tree/releases/v1) action or the [Azure/k8s-set-context](https://github.com/Azure/k8s-set-context/tree/releases/v1) action.
 
-The definition of this Github Action is in [action.yml](https://github.com/Azure/webapps-deploy/blob/master/action.yml).
+The definition of this Github Action is in [action.yml](https://github.com/Azure/webapps-deploy/blob/master/action.yml). Currently startup-command is not supported for Windows Web Apps and for Publish Profile auth scheme.
```
[the remark this answers is not in the document]
Comment (aksm-ms) [reply README.md:13]: for windows + SPN we dont support. In general, windows apps we dont support currently.

[the remark this answers is not in the document]
Comment (N-Usha) [reply README.md:13]: okk.. lets just call out what we support then 👍

File: README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ You can also use this Github Action to deploy your customized image into an Azur
 
 For deploying container images to Kubernetes, consider using [Kubernetes deploy](https://github.com/Azure/k8s-deploy) action. This action requires that the cluster context be set earlier in the workflow by using either the [Azure/aks-set-context](https://github.com/Azure/aks-set-context/tree/releases/v1) action or the [Azure/k8s-set-context](https://github.com/Azure/k8s-set-context/tree/releases/v1) action.
 
-The definition of this Github Action is in [action.yml](https://github.com/Azure/webapps-deploy/blob/master/action.yml).
+The definition of this Github Action is in [action.yml](https://github.com/Azure/webapps-deploy/blob/master/action.yml). Currently startup-command is supported for Linux Apps and not for Windows Apps. Also, startup-command is not supported for Publish Profile auth scheme.
```
Comment (N-Usha) [inline README.md:13]: I would rephrase this as "Start-up command is applicable only for Linux apps and not for Windows apps. Currently startup command is supported only for Linux apps when SPN is provided. And not when publish profile is provided."

Comment (aksm-ms) [reply README.md]: Updated.
